dev/PipsControl/PipsControl.xaml

```diff
@@ -0,0 +1,105 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls">
+
+
+    <!-- Explain why background should be set for pointer events -->
+    <Style TargetType="muxc:PipsControl">
+        <Setter Property="Background" Value="Transparent"/>
+        <Setter Property="HorizontalAlignment" Value="Left"/>
+        <Setter Property="VerticalAlignment" Value="Top"/>
+        <Setter Property="PreviousButtonStyle" Value="{StaticResource PipsControlPreviousPageButtonStyle}"/>
+        <Setter Property="NextButtonStyle" Value="{StaticResource PipsControlNextPageButtonStyle}"/>
+        <Setter Property="SelectedIndicatorStyle" Value="{StaticResource PipsControlSelectedPipButtonStyle}"/>
+        <Setter Property="DefaultIndicatorStyle" Value="{StaticResource PipsControlDefaultPipButtonStyle}"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="muxc:PipsControl">
+                    <Grid x:Name="RootGrid"
```

Comment:
>Grid [](start = 21, length = 4)

should this be a stack panel? it would make changing the orientation much easier

Reply:
StackPanel takes all the space it gets - not the space it needs. I chose Grid because with properly set Horizontal and Vertical alignments - Grid takes only the needed space. If you have an idea how to get the same behavior with a Stack Panel, please let me know :)